code/game/machinery/vending.dm

```diff
@@ -927,6 +927,21 @@
 	contraband = list(/obj/item/weapon/stock_parts/cell/potato = 3)
 	premium = list(/obj/item/weapon/storage/belt/utility = 3)
 
+/obj/machinery/vending/liberationstation
+	name = "\improper Liberation Station"
+	desc = "An overwhelming amount of ancient patriotism washes over you just by looking at the machine."

```

Comment:
"Ancient patriotism" should be bold and span class userdanger to get that extra freedom into the description